package.json

```diff
@@ -39,7 +42,7 @@
   ],
   "icon": "icon/openshift_logo.png",
   "engines": {
-    "vscode": "^1.49.0"
+    "vscode": "^1.50.0"
```

Comment:
What is the reason for bumping the version?

Reply:
to get it in sync with https://github.com/redhat-developer/vscode-commons/blob/master/package.json#L17